src/main/resources/content/RFI/rfiType.json

```diff
@@ -0,0 +1,52 @@
+{
+    "breadCrumbs": [
+        {
+            "text": "Home",
+            "href": "/"
+        },
+        {
+            "text": "Contract award service",
+            "href": "#"
+        },
+        {
+            "text": "How will you publish your RfI",
+            "href": "#"
+        }
+
+    ],
+
+    "title": "How will you build your Request for Information (RfI)?",
+    "subtitle": null,
+    "form": {
+        "question": "How will you build your Request for Information (RfI)? ",
+        "radioOptions": {
+            "idPrefix": "ccs_rfi_type",
+            "name": "ccs_rfi_type",
+            "items": [
+              {
+                "value": "all_online",
+                "html": "<h3 class='govuk-heading-m'>Provide all information online</h3>",

```

Comment:
JSON File has been created for this story

Location : ccs-scale-cat-buyer-ui/src/main/resources/content/RFI/rfiType.json

i have reviewed the code please remove the html content from JSON and utilise JSON only for the content and add the HTML Markups in the .njk files.